notebooker/constants.py

```diff
@@ -128,8 +128,10 @@ class NotebookResultError(NotebookResultBase):
 
     @property
     def raw_html(self):
-        return """<p>This job resulted in an error: <br/><code style="white-space: pre-wrap;">{}</code></p>""".format(
-            self.error_info
+        formatted_stdout = "<br/>".join(self.stdout) if self.stdout else ""
```

Comment:
Do you have a screenshot of this working? I had a suspicion that self.stdout wasn't populated for errors for some reason.